Calendar/BookingHelpers/BookingTenantSettingsHelper.ps1

```diff
@@ -0,0 +1,106 @@
+﻿function Get-BookingTenantSettings {
+    param([string] $domain)
+
+    if ($Script:MSSupport) {
+        $script:OrgConfig = Get-OrganizationConfig -Organization $domain
+        $script:OWAMBPolicy = Get-OwaMailboxPolicy -Organization $domain
+        $script:AcceptedDomains = Get-AcceptedDomain -Organization $domain
+    } else {
+        $script:OrgConfig = Get-OrganizationConfig
+        $script:OWAMBPolicy = Get-OwaMailboxPolicy
+        $script:AcceptedDomains = Get-AcceptedDomain
+    }
+    $ewsSettings = Get-EWSSettings $script:OrgConfig
+    $bookingsSettings = Get-BookingsSettings $script:OrgConfig
+    $OWAMBPolicy = Get-OwaMBPolicy $script:OWAMBPolicy
+    $acceptedDomains = Get-AcceptedDomains $script:AcceptedDomains
+    # Define the structure of the tenant settings
+    $tenantSettings = [PSCustomObject]@{
+        Identity         = $org.Identity
+        Guid             = $org.Guid
+        DisplayName      = $org.DisplayName
+        IsDeHydrated     = $org.IsDeHydrated
+        EWSSettings      = $ewsSettings
+        BookingsSettings = $bookingsSettings
+        OWAMBPolicy      = $OWAMBPolicy
+        AcceptedDomains  = $acceptedDomains
+    }
+
+    # Return the tenant settings
+    return $tenantSettings
+}
+
+function Get-EWSSettings {
```

Comment:
>Get-EWSSettings [](http://example.com/codeflow?start=9&length=15)

I would remove the "-" from function names so they are easier to tell apart from cmdlets.

Reply:
Doing that ASAP